docs/source/notebooks/finch-usage.ipynb

```diff
@@ -558,10 +558,10 @@
        "Attributes:\n",
        "    units:          days\n",
        "    cell_methods:   time: minimum (interval: 30 minutes) time: minimum within...\n",
-       "    xclim_history:  tas=max(195,tas) applied to raw data; min of 194.73 detec...\n",
+       "    history:        tas=max(195,tas) applied to raw data; min of 194.73 detec...\n",
        "    standard_name:  days_with_air_temperature_below_threshold\n",
-       "    long_name:      Number of frost days (tmin &lt; 0c)\n",
-       "    description:    Annual number of days with minimum daily temperature belo...</pre><div class='xr-wrap' hidden><div class='xr-header'><div class='xr-obj-type'>xarray.DataArray</div><div class='xr-array-name'>'frost_days'</div><ul class='xr-dim-list'><li><span class='xr-has-index'>time</span>: 20</li><li><span class='xr-has-index'>lat</span>: 6</li><li><span class='xr-has-index'>lon</span>: 7</li></ul></div><ul class='xr-sections'><li class='xr-section-item'><div class='xr-array-wrap'><input id='section-3fdbff45-0f19-4cf0-9bd4-9c79128fee37' class='xr-array-in' type='checkbox' checked><label for='section-3fdbff45-0f19-4cf0-9bd4-9c79128fee37' title='Show/hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-array-preview xr-preview'><span>122.0 121.0 98.0 47.0 33.0 19.0 ... 249.0 229.0 193.0 156.0 103.0</span></div><div class='xr-array-data'><pre>array([[[122., 121., ...,  19.,   9.],\n",
+       "    long_name:      Number of frost days (tmin &lt; 0 degc)\n",
```

Comment:
This change also breaks Jenkinks, http://jenkins.ouranos.ca/job/PAVICS-e2e-workflow-tests/job/master/1309/console

Same for the `xclim_history:` to `history:`.